app/javascript/components/users/SignInComponent.tsx

```diff
@@ -57,47 +78,186 @@ function SignInComponent(props:SignInComponentProps) : JSX.Element {
+			padding: theme.spacing(2),
+			borderRadius: 15,
+		},
+		box: {
+			justify: "center",
+			alignContent: "center",
+			display: "flex",
+		},
+	}),
+	);
 
-	// TODO
-	// const validationSchema = yup.object({
-	// 	email: yup.string().required().email(),
-	// 	// TODO:
-	// });
+	const Button = styled(MuiButton)(spacing);
+	const classes = useStyles();
 
 	return (
-		<Formik initialValues={{email: ""}}  onSubmit={async (_values, formikHelpers) => {
-			try {
-				await signIn({email: 'email@ema.com', password: "password"});
-			}
-			catch (e:unknown) {
-				// NOTE: We're just swallowing the exception here for now. Might we need to do
-				// something different? Don't know!
-			}
-			finally {
-				formikHelpers.setSubmitting(false);
+		<Formik
+			initialValues={
+				{
+					email: "",
+					password: "",
+				}}
+			validationSchema={validationSchema}
+			onSubmit={async (values, formikHelpers) => {
+				try {
+					await signIn(values);
+				}
+				catch (e: unknown) {
+					// NOTE: We're just swallowing the exception here for now. Might we need to do
+					// something different? Don't know!
+				}
+				finally {
+					formikHelpers.setSubmitting(false);
+				}
 			}
-		}
-		}>{(props) => {
+			}>{({ errors, isValid, touched, handleChange }) => {
 				useEffect(() => {
-					setIsValid(props.isValid);
-				}, [props.isValid]);
-
-				return <Form>
-					{/* NOTE: if a Button should submit a form, mark it as type="submit". Otherwise pressing Enter won't submit form*/}
-					<Button data-testid="signInButton" type="submit"
-						variant={'contained'}
-						color={'primary'}>{formatMessage({id: 'hello'})}</Button>
-
-					{componentState === 'submitting' ? "" : <>
-						<div data-testid="signInErrorDiv">{ failed ? lastError.data.error.map((i) => i).join('; ') : ""}</div>
-						<div data-testid="currentUserDiv">{currentUser ? currentUser.id : ""}</div>
-					</>
-					}
-
-				</Form>;
-			}}
+					setIsValid(isValid);
+				}, [isValid]);
+				return (
+					<Form>
+						{/* NOTE: if a Button should submit a form, mark it as type="submit". Otherwise pressing Enter won't submit form*/}
+						<Grid container
+							direction="column"
+							alignItems="center"
+							justify="center"
+						>
 
+							<Paper className={classes.paper} elevation={6}>
+								<Grid container wrap="nowrap" spacing={2}>
+									<Grid item
+										direction="column"
+										alignItems="center"
+										justify="center"
+									>
+										<CardMedia component="img"
+											src={logo}
+											title="Houdini"
+										/>
+
+										<Box p={3} display="flex" justifyContent="center" alignItems="center">
+											<Typography gutterBottom variant="h5" component="h2">
+											Login
+											</Typography>
+										</Box>
+										<Box display="flex" justifyContent="center" alignItems="center">
+											<Box p={1.5}>
+												<InputLabel htmlFor="email">Email</InputLabel>
+												<TextField
```

Comment:
As a reminder, use the `Field` from formik and then pass TextField as the component as shown at https://stackworx.github.io/formik-material-ui/docs/api/material-ui/#textfield. Formik Field component handles a number of parts of this, including setting the onChange function, allowing you to set the label so that errors have the proper field names.